google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITObjectTest.java

```diff
@@ -149,95 +196,126 @@ public void testCreateBlob() {
   public void testCreateBlobMd5Crc32cFromHexString() {
     String blobName = "test-create-blob-md5-crc32c-from-hex-string";
     BlobInfo blob =
-        BlobInfo.newBuilder(BUCKET, blobName)
+        BlobInfo.newBuilder(bucketFixture.getBucketInfo().getName(), blobName)
```

Comment:
There's an overload on this method where you can pass it the `BucketInfo` and it'll extract the name.
`BlobInfo.newBuilder(bucketFixture.getBucketInfo(), blobName)`